app/nostra/account/csv_data_parser.rb

```diff
@@ -0,0 +1,79 @@
+module Nostra
+  module CSVDataParser
+    def self.transactions(csv_file)
+      transactions_data = parse_csv_data(csv_file)
+    end
+
+    private
+
+    def self.parse_csv_data(csv_file)
+      csv_data = strip_bom_characters(csv_file)
+      transaction_rows = collect_transactions(csv_data)[1..-1]
+
+      transactions_data = []
+
+      transaction_rows.each do |transaction|
+        fields = fields_for_transaction(transaction)
+
+        # NOTE: amount_1 is what we want for the amount
+        # I'm not sure what amount_2 is in 1Money, but they don't seem to differ
+        transaction_data =   {
+          date: fields[:date],
+          type: fields[:type].downcase.to_sym,
+          from_account: fields[:from_account],
+          to_account: fields[:to_account],
+          amount: fields[:amount_1],
+          amount_2: fields[:amount_2],
+          notes: fields[:notes]
+        }
+
+        unless fields[:type] == "Transfer"
+          transaction_data.merge!({category: fields[:to_account]})
+        end
+
+        transactions_data << transaction_data
```

Comment:
might be nice to distinguish between these two variables a bit more, just the `s` is a bit hard to notice

Reply:
I agree, will probably rename